service/src/test/java/uk/gov/hmcts/reform/fpl/service/NoticeOfProceedingsServiceTest.java

```diff
@@ -61,8 +61,7 @@
     @Autowired
     private HearingVenueLookUpService hearingVenueLookUpService;
 
-    private CommonCaseDataExtractionService commonCaseDataExtractionService = new CommonCaseDataExtractionService(
-        hearingVenueLookUpService);
+    private CommonCaseDataExtractionService commonCaseDataExtractionService = new CommonCaseDataExtractionService();
```

Comment:
Could just be autowired in

Reply:
Although I guess just creating a new one doesn't actually hurt and requires less from spring so may be slightly better in performance

Reply:
will leave as is